blocks/api/serializer.js

```diff
@@ -187,8 +187,13 @@ export function serializeBlock( block ) {
 
 	switch ( blockName ) {
 		case 'core/more':
-			const { text, noTeaser } = saveAttributes;
-			return `<!--more${ text ? ` ${ text }` : '' }-->${ noTeaser ? '\n<!--noteaser-->' : '' }`;
+			const { customText, noTeaser } = saveAttributes;
+			saveContent = `<!--more${ customText ? ' ' + customText : '' }-->`;
+			if ( noTeaser ) {
+				saveContent += '\n<!--noteaser-->';
+			}
+
+			return saveContent;
```

Comment:
was there any real change here other than the name of `text` vs `customText`? I ask because we changed the declarative return with an imperative mutating return :wink:

Reply:
The only effective difference should be the change in name from `text` to `customText` yes. I thought I might have made an improvement by refactoring the template+ternary soup but... well, we've been around this block before and I really don't care enough one way or the other to argue it 😄

Reply:
I won't gripe if you make a change to prefer your style over mine in this case, but maybe there's a middle-ground.

```js
const moreTag = text
	? `<!--more ${ text }-->`
	: '<!--more-->';

const noTeaserTag = noTeaser
	? '<!--noteaser-->'
	: '';

return compact( [ moreTag, noTeaserTag ] ).join( '\n' );
```

I'm a huge fan of ternaries (they are _expressions_ and not _statements_ like their cousins, the `if` structure) but I _do_ draw the line at nesting ternaries.

Reply:
>maybe there's a middle-ground.

Looks good to me. Updated in rebased 3ba64768ec95f58ed7dcdb4f576e486cba76fa3c .